pathfinder/paizo/player_companion/blood_of_the_elements/_blood_of_the_elements.pcc

```diff
@@ -0,0 +1,48 @@
+# CVS $Revision$ $Author$ -- Tue Dec 15 01:48:08 2015 -- reformated by PCGen PrettyLST v6.06.00
+CAMPAIGN:Blood of the Elements
+GAMEMODE:Pathfinder
+RANK:9
```

Comment:
IIRC RANK should be 2 (Primary publisher of the game system is 1 is Core, 2 is Supplement, 3 is Campaign) 
9 would be extremely low homebrew, which this is not.